packages/design-system-react-native/scripts/build.js

```diff
@@ -10,6 +10,42 @@ try {
   console.log('Step 3: Building the project...');
```

Comment:
This needs to be moved to a separate PR but should fix this issue https://github.com/MetaMask/metamask-design-system/issues/652